### ISSUE TITLE
Publishing .net5 winform app gets  "Invalid input specification: Found multiple entries with the same BundleRelativePath"

### ISSUE THREAD

**Issue by xiaoyuvax** [dotnet/sdk#16195]
# Publishing .net5 winform app to single file gets  "Found multiple entries with the same BundleRelativePath"

Seen while publishing  with options of:
.net5 
self-contained
win-x64 
single file 
ReadyToRun

... can't get a clue where multiple entries are, not a hint.

Console output:
15> “GenerateBundle” task failed unexpectedly.
15>System.ArgumentException: Invalid input specification: Found multiple entries with the same BundleRelativePath
15>   in Microsoft.NET.HostModel.Bundle.Bundler.GenerateBundle(IReadOnlyList`1 fileSpecs)
15>   in Microsoft.NET.Build.Tasks.GenerateBundle.ExecuteCore()
15>   in Microsoft.NET.Build.Tasks.TaskBase.Execute()
15>   in Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
15>   in Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()

Publish without "single file" option would not produce this exception.

**Comment by mc0re** [comment]
It would help if at least the conflicting paths were given in the error.

**Comment by gcaughey** [comment]
It would help even more if the output indicated how to fix this. I'm seeing this on a project that I can publish fine as framework depending, not single file. The issue seems to be with self-contained publish.

**Comment by cosmo0** [comment]
> It would help even more if the output indicated how to fix this.

YES! As it stands, the message is about as useful as "an error occurred". If you know that entries have the same path, tell me the path!

**Comment by ThadHouse** [comment]
Any update to this? Getting this trying to build a pretty simple project.
